src/Components/DateInputForm/DateInputForm.tsx

```diff
@@ -1,22 +1,37 @@
-import React from "react";
+import React, {FormEvent, SetStateAction, useState} from "react";
 import "./DateInputForm.scss"
+import {getRoverImages} from "../ApiFetch/RoversApiFetch";
+import {RoverGalleryImage} from "../RoverGalleryImage/RoverGalleryImage";
 
 
-export function DateInputForm() {
+interface DateInputFormProps {
+    dateInput : string | null;
+    setDateInput : (date: string) => void;
+}
+
+export function DateInputForm(props : DateInputFormProps) {
+    const [date, setDate] = useState("");
+    
     function getDateToday(){
         let today: string = new Date().toISOString().split("T")[0];
         return today;
     }
+    
+    function handleClick(event: FormEvent) {
+        console.log(date);
```

Comment:
I wouldn't check console.log()s into master - that way you can end up with your console filling up with a whole load of fairly meaningless logs as the project goes on, which makes debugging harder because you have to sift through them all to find what you're looking for. I assume you did this for testing your own code anyway and just forgot to delete it.

Reply:
Thanks Steve! deleted.